src/sections/footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./Footer.css";
+import { ICONS } from "@/utils/icons";
+import Image from "next/image";
+
+
+const footer = () => {
+  return (
+    <div>
+      <div className="backgroundforfooter">
+  {/* Main content */}
+  <div className="maincolumn">
+    {/* First column */}
+    <div className="firstcolumn">
+      
+    <img src="/assets/ARS_1_Grey_cropped.png"  alt="Captcha"/>
```

Comment:
**Suggestion:** If possible, use `<Image />` instead of `<img>`. *(Not mandatory)*